### ISSUE TITLE
Unable to resolve path to module 'async_hooks'

### ISSUE THREAD

**Issue by smolyakoff** [benmosher/eslint-plugin-import#996]
Eslint gives an unexpected error when `async_hooks` core Node.js module is required: 
```
Unable to resolve path to module 'async_hooks'
```
See related issue at https://github.com/browserify/resolve/issues/144

**Comment by ljharb** [comment]
This will be addressed once `resolve` gets an updated release; there's no need to change anything here.
